lib/stdlib/test/ets_SUITE.erl

```diff
@@ -9061,6 +9064,17 @@ ets_apply(F, Args, Opts) ->
 ets_format_args(Args) ->
     lists:join(", ", [io_lib:format("~p", [A]) || A <- Args]).
 
+bound_maps(_Config) ->
+    T = ets:new('__bound_maps__', [ordered_set, public]),
+    Ref = make_ref(),
+    Attrs = [#{}, #{key => value}],
+    [ets:insert_new(T, {{Attr, Ref}, original}) || Attr <- Attrs],
+    Attr = #{},
+    Key = {Attr, Ref},
+    MS = [{{Key, '$1'},[],[{{{element, 1, '$_'}, {const, new}}}]}],
+    2 = ets:select_replace(T, MS),
```

Comment:
This tests only the `ordered_map` (original bug was discovered using `set`). I understand that with ordered_map it is easier to reproduce, but I wonder if the test case should also do `set`. You know, just in case, if implementation diverges.

Reply:
`set` often passes by accident, and I wanted to avoid a flaky tests as both had the same root cause. We can expand the test on `master` once 27 is out. Thanks :)